data/categories.txt

```diff
@@ -1653,9 +1669,10 @@ fa:مغازه کادو فروشی
 
 shop-beauty
 en:4Beauty Shop|barber|beautician|hairdresser|hairdressing|haircut|U+1F484|beauty salon|hair salon|beauty parlor|hair and nail salon|nail salon|coloring
-ru:5Салон красоты|косметический салон|парикмахерская|салоны красоты и парикмахерские|стрижка|подстричься|маникюр|маникюрный салон|красота|студия красоты|центр красоты
-bg:5Салон за красота|Фризьорски салон|Козметичен салон|Маникюр|Педикюр|подстригване|кола маска
+ru:5Салон красоты|косметический салон|парикмахерская|стрижка|маникюр|маникюрный салон|красота|студия красоты|центр красоты
 ar:صالون تجميل|كوافير|متجر
+be:5Салон прыгажосці
+bg:5Салон за красота|Фризьорски салон|Козметичен салон|Маникюр|Педикюр|подстригване|кола маска
 cs:4Kosmetický salon
 da:Skønhedssalon
 nl:Schoonheidssalon
```

Comment:
nl:Schoonheidssalon|kapper|nagelstudio

Isn't barber a separate category?

Reply:
Yes, but the term may be used for many times, no problem here.

I suppose that you can get a hair cut in a beauty shop too (at least in 90%) :)